lists.js

```diff
@@ -398,6 +398,22 @@ List.prototype.becomeLinked = function () {
     }
 };
 
+List.prototype.asJSON = function () {
```

Comment:
Why can't we define a `.toJSON` method and then just call `JSON.stringify()`?

Though, I don't know how that would work for list items that can't be easily serialized to JSON...